install-orcsdr.ps1

```
@@ -1,189 +1,73 @@
+  $env:IDF_PYTHON_ENV_PATH = 'C:\Espressif\python_env\idf5.5_py3.14_env'; $env:PATH = "C:\Espressif\tools\ccache\4.12.1\ccache-4.12.1-windows-x86_64;$env:IDF_PYTHON_ENV_PATH\Scripts;$env:PATH"
+  . (Join-Path $IdfPath 'export.ps1')
 }
-
-function Test-HostedPair {
-  param(
-    [Parameter(Mandatory = $true)][string]$ComPort,
-    [int]$Seconds = 25
-  )
-  $serial = $null
-  $saw = $null
+function Flash-App([string]$App, [string]$Build, [string]$ComPort) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Flash-App' uses an unapproved verb.

(PSUseApprovedVerbs)